docs/install-upgrade/install.md

```diff
@@ -41,19 +41,29 @@ Hedgehog has created a command line utility, called `hhfab`, that helps generate
 1. The `init` command generates a `fab.yaml` file, edit the `fab.yaml` file for your needs
     1. ensure the correct boot disk (e.g. `/dev/sda`) and control node NIC names are supplied
 1. `hhfab validate`
-1. `hhfab build`
+1. `hhfab build --mode iso`
+    1. There are two options for the format of the bootable media. ISO and IMG. An ISO is best suited to use with IPMI based virtual media. The img file is designed for writing to a removable USB drive.
 
-The installer for the fabric is generated in `$CWD/result/`. This installation image is named `control-1-install-usb.img` and is 7.5 GB in size. Once the image is created, you can write it to a USB drive, or mount it via virtual media.
+The installer for the fabric is generated in `$CWD/result/`. This installation image is named `control-1-install-usb.iso` and is 7.5 GB in size. Once the image is created, you can write it to a USB drive, or mount it via virtual media.
```

Comment:
Shouldn't you be using the `img` format for writing to USB?

Reply:
yes, yes you should.

Reply:
What I meant is that the sentence says you generate a `.iso` file and then you can write it to a USB drive. It should probably only mention the virtual media mounting, _or_ stipulate that the image can also have a `.img` extension, and in the latter case it can be used with a USB drive.